File: vcc-sdk-java/src/main/java/com/ctrip/ccard/creditcard/vcc/bean/MerchantInfo.java
package com.ctrip.ccard.creditcard.vcc.bean;

import java.util.List;

/**
 * Description:
 */
public class MerchantInfo {

    /**
     * 商户名称
     */
    private String merchantName;

    /**
     * 支持的通道信息列表
     */
    public List<ChannelInfo> channelInfos;

    public String getMerchantName() {
        return merchantName;
    }

    public void setMerchantName(String merchantName) {
        this.merchantName = merchantName;
    }

    public List<ChannelInfo> getChannelInfos() {
        return channelInfos;
    }

    public void setChannelInfos(List<ChannelInfo> channelInfos) {
        this.channelInfos = channelInfos;
    }
}

File: php-sdk/src/v1/client/query/MerchantInfo.php
<?php
/**
 * Created by PhpStorm.
 * User: wang.xiaoliang
 * Date: 2021/8/4
 * Time: 11:17
 */

namespace triplink\vcc\card\client\query;

class MerchantInfo {
    /**
     * 商户名称
     */
    var $merchantName;

    /**
     * 支持的通道信息列表
     */
    var $channelInfos;

    /**
     * @return mixed
     */
    public function getMerchantName()
    {
        return $this->merchantName;
    }

    /**
     * @param mixed $merchantName
     */
    public function setMerchantName($merchantName): void
    {
        $this->merchantName = $merchantName;
    }

    /**
     * @return mixed
     */
    public function getChannelInfos()
    {
        return $this->channelInfos;
    }

    /**
     * @param mixed $channelInfos
     */
    public function setChannelInfos($channelInfos): void
    {
        $this->channelInfos = $channelInfos;
    }



}
?>